.circleci/config.yml

```diff
@@ -496,7 +496,7 @@ jobs:
 
   packaging-linux-baobab:
     machine:
-      image: ubuntu-2004:202201-02
+      image: klaytn/build_base:1.2-ubuntu18.04-go.1.20.6-solc0.8.13
```

Comment:
It is not match with `packaging-linux`.
```
    docker:
      - image: klaytn/build_base:1.2-ubuntu18.04-go.1.20.6-solc0.8.13
```

Reply:
I'm confuse in here, could you explain more pls?

Reply:
Oh Sorry it was my fault. I have updated it. Ignore this comment :)